src/style.css

```diff
@@ -45,4 +45,9 @@ header {
   transform: translateY(-0.25em) translateX(0.25em);
 }
 
+.button-heavy-shadow:active{
+  background-color:darkgray;
+  color: white;
+}
+
```

Comment:
looks good! Just check next time that you merge into the correct branch. you wanted to merge changes into the main. I changed it to dev